LICENSE

```diff
@@ -199,68 +199,3 @@
    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
    See the License for the specific language governing permissions and
    limitations under the License.
-
```

Comment:
The rest of this file is copied from Apache Spark repository and is invalid in this Swift repository.